pages/form/Question2.jsx

```diff
@@ -1,78 +1,126 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
 import {
-  Box,
-  Grid,
+  Wrap,
+  WrapItem,
+  Button,
   Heading,
   Text,
-  Select,
+  Radio,
+  RadioGroup,
+  Collapse,
 } from "@chakra-ui/react";
 import Layout from "../../components/Layout/Layout";
 import useForm from "../../components/FormProvider";
-import { modesOfTransport } from "../../utils/constants";
+import { daysOfWeek, modesOfTransport } from "../../utils/constants";
 import Q2Progress from "../../public/images/progress-bar/q2-progress-dots.svg";
-import { BackButton, ContinueButton } from "../../components/LinkButton/LinkButton";
+import Q2Cloud from "../../public/images/clouds/cloud-q2.svg";
+import LinkButton, { BackButton, ContinueButton } from "../../components/LinkButton/LinkButton";
+
+const WFH = "work from home";
+const ON_SITE = "on-site";
 
 export default function Question2() {
   const { answers, setAnswers } = useForm();
-  const [selectedMode, setSelectedMode] = useState(answers.mainTransportMode || "");
+  const [daysSelected, setDaysSelected] = useState([]);
+  const [workMode, setWorkMode] = useState(answers.workMode);
```

Comment:
@ymloh ,
what does `workMode` capture, and what are its possible values?

Reply:
ok, I think `workMode` captures whether user selected 'I work fully from home' (`WFH`), or 'I work on-site' (`ON_SITE`) ? I that right?

Reply:
`workMode` stores the value of the radio button selected, which are the constants `WFH` and `ON_SITE`. I actually hadn't considered that this might end up on the Google Sheet as well; perhaps we should use some other context to store this?